src/components/NewPerson/NewPerson.tsx

```diff
@@ -0,0 +1,50 @@
+import classNames from 'classnames';
+import { Person } from '../../types/Person';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  person: Person;
+  selectedPerson: string | undefined;
+};
+
+export const NewPerson: React.FC<Props> = ({ person, selectedPerson }) => {
```

Comment:
```suggestion
export const Person: React.FC<Props> = ({ person, selectedPerson }) => {
```

Reply:
I am unable to use this particular name as it conflicts with my interface's name. In order to avoid any import conflicts, I have renamed it to PersonItem.